KenticoOnboardingApplication/test/KenticoOnboardingApplication.Api.Tests/App_Start/DependencyConfigTests.cs

```diff
@@ -57,9 +60,8 @@ private static List<Type> GetExpectedRegistrationTypes()
                 .GetTypes()
                 .Where(type => type.IsInterface && !s_excludedContractsTypes.Contains(type))
                 .ToList();
-            expectedContractsTypes.AddRange(s_expectedRegistrationTypes);
 
-            return expectedContractsTypes;
+            return expectedContractsTypes.Union(s_expectedRegistrationTypes).ToList();
```

Comment:
[👎](https://tenor.com/view/oh-fuck-gif-6152513) Have you been in a hurry? 😆 
```csharp
            var expectedContractsTypes = contractsAssembly
                .GetTypes()
                .Where(type => type.IsInterface && !s_excludedContractsTypes.Contains(type))
                .Union(s_expectedRegistrationTypes)
                .ToList();
```
(there is no big perfermance hit here, yet there is not art no finesse in that! 😉)